composer.json

```diff
@@ -66,6 +66,11 @@
     }
   ],
   "scripts": {
-    "lint": "phpcs --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1"
```

Comment:
Can we keep the `lint` script? This is not used yet, but eventually – when proper coding standards finally come 😅 – it might be useful.

Reply:
Sure! I would love Timber to have coding standards (please don't use the WordPress ones) and mostly a fixer command so I don't have to care about CS anymore 😁